irma-frontend/package.json

```diff
@@ -9,7 +9,9 @@
     "release": "npm run build"
   },
   "license": "MIT",
-  "browsersList": "> 0.25%, not dead",
+  "browserslist": [
+    "> 0.25%, not dead"
+  ],
```

Comment:
Does this also need to be fixed in the root `package.json`?

Reply:
Oh weird, then I know why I made this mistake :). I'll fix it there too. That one is only for the linter.